ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -169,6 +169,11 @@ object Cli {
            |Party names must be their hints.""".stripMargin)
 
     opt[Unit]("list")
+      .action((_, c) => c.copy(listTestSuites = true))
+      .text(
+        """Lists all available test suites that can be used in the include and exclude options. Test names always start with their suite name, so using the suite name as a prefix matches all tests in a given suite.""")
+
+    opt[Unit]("list-all")
```

Comment:
Happy to leave this up to you but an alternative UX would be `list --all`. Not sure what I prefer.

Reply:
I agree the `--list`/`--list-all` thing is a bit awkward, but I believe fixing this should be done as part of a broader redesign of the CLI; for example the situation with perf tests vs. normal tests could also use a more explicit structure. I don't want to extend the scope of this PR though.